build.gradle

```diff
@@ -77,6 +79,37 @@ subprojects {
         }
     }
 
+    plugins.withId('maven-publish') {
+        publishing {
+            publications {
+                nebula(MavenPublication)
+            }
+            repositories {
+                maven {
+                    name = 'Milestone'
+                    url = 'https://repo.spring.io/milestone'
```

Comment:
I don't know the difference between using the maven-publish plugin directly versus using the artifactory plugin to publish to this repository. The snapshot publish uses the artifactory plugin as far as I understand. Could we check and compare a snapshot vs candidate release with these changes?

Reply:
Since it is just a maven repo eventually, both will work, the artifactory plugin was a little more convenient since it was already set-up. Though, using the same process for all of our repos would make more sense to me too, also it will enable us to remove the artifactory plugin. I added the snapshot repo, the artifactory plugin is not used from now on and everything is published in the same way.